### PR TITLE
Add Fire Emblem Echoes: Shadows of Valentia

### PR DESCRIPTION
**Pull request by Kampfarsch**
Adds Fire Emblem Echoes: Shadows of Valentia.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by BreadFish64** [comment]
the boxart should have the 3DS logo on the side

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title = "Fire Emblem Echoes: Shadows of Valentia"
+description = "Fire Emblem Echoes: Shadow of Valentia\nIn a continent that was divided by warring gods, guide legions of soldiers through tactical battles—as this reimagined RPG epic invades the West!\n\nUncover the intertwining tales of heroes Alm and Celica, engage foes, fend off monsters, and explore 3D dungeons. Your prowess will grow as you master this melding of modern and legacy Fire Emblem gameplay."
```
Comment (LCypher00) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Get rid of those "\n" 's everywhere.
I would advise you to try and write your own description ...
Also the description should include the publisher and genre of the game.
Please refer to this guide for further information : https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "00040000000B8B00"
```
Comment (LCypher00) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: This is not the right title id.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+region = "usa"
+release_date = "2017-05-19"
+
+[[releases]]
+title = "00040000001B4100"
+region = "eur"
+release_date = "2017-05-19"
+
```
Comment (LCypher00) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: I believe the game also released in japan you should add it here.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat (hunk 1 above)
Comment (LCypher00) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: The striped lines glitch is a known issue : https://github.com/citra-emu/citra/issues/2846 , it should be mentioned here under description (refer to the guide linked above)

Comment (Leo626) [reply games/fire-emblem-echoes-shadows-of-valentia/game.dat]: The description should be short and can pulled from the game's Wikipedia page (minus the release dates).

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+[[testcases]]
+title = "00040000001B4100"
+compatibility = "3"
+date = "2017-08-31"
+version = "HEAD-e450a2d"
+cpu = "Intel Core  i5-6500"
```
Comment (TheKoopaKingdom) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: There's an extra space after `Core` here.

**Comment by Schplee** [comment]
PR author has apparently abandoned this. Closing.

**Pull request closed without merging**